apolloschurchapp/android/app/src/main/AndroidManifest.xml

```diff
@@ -14,21 +14,30 @@
       android:roundIcon="@mipmap/ic_launcher_round"
       android:allowBackup="false"
       android:theme="@style/AppTheme">
+
       <meta-data
         android:name="com.google.android.geo.API_KEY"
         android:value="@string/GOOGLE_MAPS_API_KEY"/>
+
       <activity
         android:name=".MainActivity"
         android:label="@string/app_name"
         android:configChanges="keyboard|keyboardHidden|orientation|screenSize"
         android:windowSoftInputMode="adjustPan"
+        android:exported="true"
+        android:launchMode="singleTask">
```

Comment:
Does this have any side affects we know of?

Reply:
So `launchMode` I called out in the PR description. We had it before and will continue to need it. I updated our usage to match what this package says we'll need for deeplinking to work. 

As for `exported` this was an add based on direction from `react-native-bootsplash`. Having just read [the documentation](https://developer.android.com/guide/topics/manifest/activity-element#exported) for it now it sounds like we probably should have already had this value set to `true` based on the suggestions in the Android docs. I think the package was just following best practices better than we were previously. To answer you question @conrad-vanl, I don't think there are any **unwanted** consequences given that we were already using `intent-filters` for deeplinks and push notifications. A positive is that now, other apps can "link into"/launch our app.